src/app/components/block-example/style.css

```diff
@@ -0,0 +1,9 @@
+.bb-route-block__example {
+	position: relative;
+	max-width: 100%;
+	box-sizing: border-box;
+}
+
+.bb-route-block__example * {
+	box-sizing: inherit;
+}
```

Comment:
this could be a linting issue (newline missing). That said, we should wait for npm to be back on track to run the actions and merge.